src/main/java/com/npsproject/it4045npsproject/It4045NpsProjectApplication.java

```diff
@@ -3,6 +3,9 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
+/**
+ * Starts the It4045NpsProject application
+ */
 @SpringBootApplication
 public class It4045NpsProjectApplication {
 
```

Comment:
Add more javadoc in increase readability and make it easier for other developers that might look at the code to understand. 
https://www.tutorialspoint.com/java/java_documentation.htm
https://blog.joda.org/2012/11/javadoc-coding-standards.html